src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1292,7 +1292,9 @@ public Class<?> getComponentType() {
         return componentType;
     }
 
-    private final Class<?> componentType;
+    // The componentType field's null value is the sole indication that the class is an array,
+    // see isArray().
+    private transient final Class<?> componentType;
```

Comment:
Why the `transient` and how does this impact serialization??

Reply:
The fields in `Class` are just inconsistently transient or not. `Class` has special treatment in the serialization specification, so the presence or absence of the `transient` modifier has no effect.

Reply:
Thanks Chen.  I was wondering why the other JVM installed fields were transient and this one wasn't so I added it to see if someone noticed and could verify whether it's right or not.